groups/migrations/0001_initial.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 4.2.6 on 2023-10-20 05:20
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Group",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created", models.DateTimeField(auto_now_add=True)),
+                ("name", models.CharField(max_length=100)),
+                ("active", models.BooleanField(default=True)),
+                (
+                    "schedule",
+                    models.CharField(
+                        choices=[
+                            ("BIWEEKLY", "Biweekly"),
+                            ("QUARTERLY", "Quarterly"),
+                            ("MONTHLY", "Monthly"),
+                        ],
+                        default="BIWEEKLY",
+                    ),
+                ),
+                ("logo_url", models.TextField()),
+                ("last_issue_date", models.DateField(blank=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Members",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "role",
+                    models.CharField(
+                        choices=[
```

Comment:
What is the difference between owner and admin?

Reply:
In my head only difference was that Owner can choose to delete a group while an admin can make changes to a group, but can't delete it.

Reply:
Yea, the only edge case i can think of is if one admin wants to delete, do we need permissions from all others or just delete because one admin said so. I'm leaning towards the latter. Cause right now there isn't really anything special that an owner should  be able to do that an admin can't

Reply:
That makes sense, I'm down to just have two roles then Admin and Member